package.json

```diff
@@ -18,6 +19,7 @@
     "path-exists": "^3.0.0",
     "pify": "^2.3.0",
     "revalidator": "^0.3.1",
+    "rimraf": "^2.5.4",
```

Comment:
Good catch. Fixed in https://github.com/zeke/electron-docs-linter/commit/5c6b2e014734819a625cc1f5771ccc2632efd2cd